pkg/pool-stable/test/ComposableStablePoolProtocolFees.test.ts

```diff
@@ -346,10 +346,6 @@ describe('ComposableStablePoolProtocolFees', () => {
 
         // The virtual supply is some factor of the invariant
         preVirtualSupply = preInvariant.mul(fp(random(1.5, 10))).div(FP_SCALING_FACTOR);
-
-        // This will store the amplification factor and invariant as the lastJoinExit values, as well as setup the
-        // old rates.
-        await pool.updatePostJoinExit(AMPLIFICATION_FACTOR, preInvariant);
```

Comment:
Note that we _do_ need for the old rates to be setup, but since they equal 1.0 and were set at construction, I think in this case it is unnecessary.

Reply:
Ah, I missed that they were being set up as well. I've re-added this line just to avoid debugging hell in future.